app/controllers/api_docs/vendor_api_docs/openapi_controller.rb

```diff
@@ -12,7 +12,7 @@ def spec_1_0
       def spec_1_1
```

Comment:
instead of tying the `spec_1_1` to draft, could we separate those and define a `spec_draft` instead to use for now?